library/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,10 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+
+    package="org.jboss.aerogear.android.unifiedpush">
+
+    <application android:allowBackup="true" android:label="@string/app_name"
+        android:supportsRtl="true">
+
+    </application>
```

Comment:
We need to original manifest, it declares services and setup that gets merged into the client app.